syft/pkg/license.go

```diff
@@ -12,6 +13,11 @@ import (
 	"github.com/anchore/syft/syft/license"
 )
 
+// FullTextValue is a placeholder value when license metadata is discovered to be fullText
+// A license with "FullText" in its value refers the consumer to look at the FullText field
+// This is so that we can keep Value as a required field up until syft 2.0
+var FullTextValue = "FullText"
```

Comment:
instead of a placeholder field that is attempting to have consumers reference the full text, should we just populate this with an empty value? That is, the json schema requires the field to be present but does not state that it must not be empty. I feel that if we don't know the ID we should leave this blank, but allow the new full text field to convey any discovered information. That way we can stay compliant to syft v1 json output but still add full text.

Reply:
When we do things like `otherLicenses` in SPDX what would we use for the license ID?

Reply:
@wagoodman - it looks like in syft we have a couple of places where we assume value should not be `""`

This includes the constructor for creating a license set:
https://github.com/anchore/syft/blob/02f9350fa53fdc8ccb3ad85bf77f103e70e1fb23/syft/pkg/license_set.go#L48-L64

If we were to keep this `const` as a placeholder some alternatives that @willmurphyscode and I discussed could be as follows:

#### Other Options for Value

- SeeFullText
- “<First_50_Characters of file> … [TRUNCATED]”

Do we want a better name for this `const` or do we want to go back and break up the license set/spdx formatter assumptions about `value` being present?

Reply:
I think this section will need to be updated already to account for filtering out "FullText" values. In that same way, can't we do the same for an empty value and the full text is specified in the other field?